.github/workflows/ci.yml

```diff
@@ -226,8 +226,8 @@ jobs:
 
     strategy:
       matrix:
-        num_runners: [2]
-        runner_index: [1, 2]
+        num_runners: [4]
+        runner_index: [1, 2, 3, 4]
```

Comment:
The fog tests are pretty resource intensive when run together
For example:
- `mc-fog-view-server::unary_smoke_tests test_view_integration::large_omap_multiple_stores` run by itself locally it takes ~30s, but often exceeds 120s in CI
- `mc-fog-sql-recovery-db::smoke_tests sql_recovery_db_smoke_tests_new_apis` run by itself locally it takes ~20s but often exceeded 60s in CI
- `mc-fog-ledger-server::router_connection fog_ledger_merkle_proofs_test` run by itself locally it takes ~13 seconds but often exceeds 60s in CI

I stopped counting after 100 (I couldn't copy and paste the threads out of the UI I was using :(), but by my estimation `mc-fog-ledger-server::router_connection fog_ledger_merkle_proofs_test` uses over 500 threads. Some of these look to be single use for gRPC and what not, but it's having to share those threads with the other fog tests that are using similar amounts of threads.